### DISCUSSION TITLE
PWAsForFirefox 2.13.2

### DISCUSSION THREAD

**Discussion by filips123** [filips123/PWAsForFirefox#601] [Announcements]
## Important

This release fixes compatibility with Firefox 134. Please make sure to update to it before Firefox 134 is released on 2025-01-07.

## Added

* Documentation about creating AppArmor profile.
* Documentation about uninstalling the project (#599).

## Changed

* Updated translations.
* Updated dependencies.

## Fixed

* Opening new window on macOS should now work again (#593, #598, thanks to @TitanNano).
* Kiosk mode should now work properly in web apps again (#176).
* The app browser should now be compatible with Firefox 134 (#600).

<hr /><em>This discussion was created from the release <a href='https://github.com/filips123/PWAsForFirefox/releases/tag/v2.13.2'>PWAsForFirefox 2.13.2</a>.</em>

**Comment by Bazoogle** [comment]
When will the update be reflected in the Firefox addon store? Currently it's still reporting the most recent version is 2.13.1

**Reply by filips123** [reply]
Probably in a few days, once the addon is approved.
